bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/LFV_TQMuTau/LFV_ST_TQMuTau/LFV_ST_TQMuTau_proc_card.dat

```diff
@@ -0,0 +1,6 @@
+import model /cms/ldap_home/ljw1015/MCProd/mastergenprod/bin/MadGraph5_aMCatNLO/SMEFT_2l2q_UFO_unitary
```

Comment:
We should add the model to our UFO path and then you can include via 
import model SMEFT_2l2q_UFO_unitary
and then just have the extra model card. Please put it to a public path so Sihyun or myself can copy.

Reply:
@agrohsje
I've uploaded the model on my cernbox.
https://cernbox.cern.ch/index.php/s/AxylpFmhS3zvoOR